src/codegen/object.rs

```diff
@@ -82,6 +82,7 @@ pub fn generate(
         } else {
             None
         },
+        None, // TODO
```

Comment:
What's to do here?

Reply:
Same, `analysis.version` :)

Reply:
See my reply there then :)

Reply:
I see no reply there?

"There" being https://github.com/gtk-rs/gir/pull/991#discussion_r526656146, not https://github.com/gtk-rs/gir/pull/991#discussion_r527488987 (I replied bottom-to-top, hence the confusion. Sorry :grimacing:)

This has anyway been fixed in the last push: https://github.com/gtk-rs/gir/compare/37af52f724ea03a12bdc6127f027333588af308e..165999fdd25bc572606975821b0fdb19a9838cdc#diff-fae3a30287d03db00601eac005d17f1a1f41a1a9d94dbef815cc26c685e037a0L85-R85

Reply:
Oh, sorry for the confusion. Please resolve discussions that are resolved :)

Reply:
Fair, perhaps now is the time to ask how you want to deal with review threads: should I mark them as resolved after fixing or notify you so that you can take another look and approve?

I personally prefer the latter (when reviewing) so that I don't have to expand all resolved side-by-side to a push-diff and doublecheck, even though the change at hand is minor enough.

Reply:
If it's something minor, just resolve it. If it's something where you're not entirely sure, keep it open :)